packages/lambda-tiler/src/routes/preview.ts

```diff
@@ -0,0 +1,157 @@
+import { ConfigTileSetRaster } from '@basemaps/config';
+import { Bounds, ImageFormat, LatLon, Projection, TileMatrixSet, VectorFormat } from '@basemaps/geo';
+import { CompositionTiff, Tiler } from '@basemaps/tiler';
+import { SharpOverlay, TileMakerSharp } from '@basemaps/tiler-sharp';
+import { HttpHeader, LambdaHttpRequest, LambdaHttpResponse } from '@linzjs/lambda';
+import { ConfigLoader } from '../util/config.loader.js';
+import { Etag } from '../util/etag.js';
+import { NotFound, NotModified } from '../util/response.js';
+import { Validate } from '../util/validate.js';
+import { DefaultBackground, DefaultResizeKernel, TileXyzRaster, isArchiveTiff } from './tile.xyz.raster.js';
+
+export interface PreviewGet {
+  Params: {
+    tileSet: string;
+    tileMatrix: string;
+    lat: string;
+    lon: string;
+    z: string;
+  };
+}
+
+const PreviewSize = { width: 1200, height: 630 };
```

Comment:
How do we know this is the size we need?

Reply:
Google says its the size of open graph images https://www.google.com/search?q=og%3Aimage+size

We could look at letting it be a customized size in the future?